### PR TITLE
ws-1216: Only include instructors for a program's courses' advertised…

### PR DESCRIPTION
**Pull request by reillz10**
… course runs

### CODE REVIEW

File: course_discovery/apps/course_metadata/tests/test_models.py [hunk 1]
```diff
@@ -1897,9 +1897,23 @@ def test_price_ranges_with_multiple_course_runs_and_none_dates(self):
         self.assertEqual(program.price_ranges, expected_price_ranges)
 
     def test_staff(self):
+        TWO_WEEKS_FROM_TODAY = datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=14)
```
Comment (rgraber) [inline course_discovery/apps/course_metadata/tests/test_models.py:1902]: can we test that we *don't* include staff from non-active course runs? since that was the problem we were originally trying to fix i think it'd be good to be explicit
also probably want to test the empty case if there's not a test for it already (no active course runs to be found)

File: course_discovery/apps/course_metadata/tests/test_models.py [hunk 2]
```diff
@@ -1897,11 +1897,39 @@ def test_price_ranges_with_multiple_course_runs_and_none_dates(self):
...
+            end=TWO_WEEKS_FROM_TODAY,
+            status=CourseRunStatus.Published,
+            enrollment_end=TWO_WEEKS_FROM_TODAY,
+            staff=set(expected_staff)
+        )
+        SeatFactory(
+            course_run=advertised_course_run,
+            type=SeatTypeFactory.verified(),
+            upgrade_deadline=TOMORROW
+        )
```
Comment (Dillon-Dumesnil) [inline course_discovery/apps/course_metadata/tests/test_models.py:1918]: Out of curiosity, why was this needed? It doesn't seem to be used in the old version of the test.

**Review by Dillon-Dumesnil — APPROVED** [review]

File: course_discovery/apps/course_metadata/tests/test_models.py (hunk 2 above)
Comment (reillz10) [reply course_discovery/apps/course_metadata/tests/test_models.py:1918]: This makes the course_run qualify as an `advertised_course_run`. If it doesn't have any seats then it can't be advertised.

**Review by rgraber — APPROVED** [review]
LGTM
